manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```
@@ -91,9 +91,8 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="cluster_version_payload"}'
     #
-    # owners: (@openshift/openshift-team-installer)
+    # owners: (@openshift/openshift-team-installer, @openshift/openshift-team-cincinnati)
```

```suggestion
    # owners: (@openshift/openshift-team-cincinnati)
```
Did you want to keep one instance of `@openshift/openshift-team-installer`?

---

Yup. Per my initial comment and the commit message:

> `cluster_installer` leans heavily on installer-supplied data, so I'm leaving that one co-owned by the installer team.